packages/client/src/runtime/query.ts

```diff
@@ -3,6 +3,7 @@ import indent from 'indent-string'
 import { bold, dim, green, red, white } from 'kleur/colors'
 import stripAnsi from 'strip-ansi'
 
+import { setClassName } from '../../../internals/dist/utils/setClassName'
```

Comment:
```suggestion
import { setClassName } from '@prisma/internals/dist/utils/setClassName'
```
Like this does not work?

Reply:
Oops, VS Code auto-insertion

Reply:
actually `import { setClassName } from '@prisma/internals'` would be the best?

Reply:
fixed